package.json

```diff
@@ -19,11 +16,10 @@
   },
   "private": true,
   "devDependencies": {
-    "acorn": "8.8.2",
+    "acorn": "8.10.0",
     "acorn-walk": "8.2.0",
-    "esbuild": "^0.17.19",
```

Comment:
not used

Reply:
Nice catch. Lots of tools come and go in the space, but last I checked https://github.com/webpro/knip is the best one at the moment for spotting these. Luckily we don't have to dig far to find these in our repos.